python/opentrons_ot3_firmware/messages/message_definitions.py

```diff
@@ -276,11 +276,10 @@ class FirmwareUpdateCompleteAcknowledge:  # noqa: D101
 class ReadLimitSwitchRequest:  # noqa: D101
     payload: payloads.EmptyPayload
     payload_type: Type[BinarySerializable] = payloads.EmptyPayload
-    message_id: MessageId.limit_sw_request
-
+    message_id: Literal[MessageId.limit_sw_request] = MessageId.limit_sw_request
```

Comment:
Sorry I missed this in the initial review. 

The significance of the `Literal[MessageId.limit_sw_request]` is to tell our type checker that `message_id` MUST be the value `MessageId.limit_sw_request`.